### PR TITLE
Create lib by poetry

### PR DESCRIPTION
**Pull request by Bruno-Felix**
## Motivation

Create and define the start of the playground library

## Changes

- added pyproject.toml poetry file

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+[tool.poetry]
+name = "my-package"
+version = "0.1.0"
+description = "A short description of the package."
+license = "MIT"
+authors = [
+    "PDA-FGA <rocha.carla@gmail.com>"
+]
+maintainers = [
+    "Bruno-Felix <auxbfelix@gmail.com>"
```
Comment (LeoSilvaGomes) [inline pyproject.toml:10]: We need to add all the maintainer here, we have a list with user and email of them, what do you think to add here?

**Comment by LeoSilvaGomes** [comment]
Please add the details about your prs, this is important for the traceability
image

**Comment by Bruno-Felix** [comment]
> Please add the details about your prs, this is important for the traceability image

Sorry. I fix this now

File: .github/workflows/python-publish.yml [hunk 2]
```diff
@@ -14,22 +14,37 @@ permissions:
...
+        with:
+          path: .venv
+          key: pydeps-${{ hashFiles('**/poetry.lock') }}
+
+      - run: poetry install --no-interaction --no-root
+        if: steps.cache-deps.outputs.cache-hit != 'true'
+
+      - run: poetry install --no-interaction
+
+      - run: poetry run pytest
```
Comment (LeoSilvaGomes) [inline .github/workflows/python-publish.yml:50]:
```suggestion
      - run: poetry run pytest --cov
```

File: .github/workflows/python-publish.yml [hunk 3]
```diff
@@ -14,22 +14,37 @@ permissions:
...
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
-    - name: Run tests
-      run: pytest --cov
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```
Comment (LeoSilvaGomes) [inline .github/workflows/python-publish.yml:24]:
```suggestion
          python-version: "3.9"
```

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+license = "MIT"
+authors = [
+    "PDA-FGA <rocha.carla@gmail.com>"
+]
+readme = "README.md"
+repository = "https://github.com/PDA-FGA/Playground"
+keywords = ["packaging", "poetry"]
+packages = [{include = "hygia", from = "src"}]
+
+[tool.poetry.dependencies]
```
Comment (LeoSilvaGomes) [inline pyproject.toml:14]: Make sure how to install and add this dependencies in the readme

**Review by LeoSilvaGomes — APPROVED** [review]

**Pull request merged**